cronjob-run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Skip opening pull requests, default: true
+export CVEJOB_SKIP_PULL_REQUESTS=${CVEJOB_SKIP_PULL_REQUESTS:-true}
+
+# Do not process CVEs older than 1 day (by default)
+export CVEJOB_CVE_AGE=${CVEJOB_CVE_AGE:-1}
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+python3.6 run.py
```

Comment:
don't we need just `python3` there? not to be tied to high to 3.6 version?

Reply:
I *think* Python 3.6 in EPEL doesn't provide python3 binary. It's already taken by "python34" package.